frontend/src/views/Queue.vue

```diff
@@ -107,16 +110,16 @@ export default {
     },
     // Incrementing Queue Length
     queueInc: function(snap) {
-      this.queueLength += snap.numChildren();
+      this.queueLength += snap.numChildren() / 2;
```

Comment:
please don't hardcode 2. Iterate through the snap and find the number of children which interests you.